main/maven/test/resources/--deps-object/pom.xml

```diff
@@ -0,0 +1,226 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Let's remove the `--` from the file name and adjust the test code accordingly. Having files named with leading `-` or `--`s often causes problems from command-line usage, e.g. `rm -rf --deps-object` may fail to delete it, and although there are workarounds its best to just avoid the problem to begin with